tests/all/test_values.rs

```diff
@@ -522,77 +525,82 @@ fn test_metadata() {
 
 #[test]
 fn test_floats() {
-    let context = Context::create();
+    #[cfg(not(any(feature = "llvm14-0", feature = "llvm15-0")))]
```

Comment:
Why exclude 14 and 15? This was running fine in 14

Reply:
Need CI for that. Fixed.